### DISCUSSION TITLE
pgRouting isn't working postgresql server 14

### DISCUSSION THREAD

**Discussion by danpanetta** [Homebrew/discussions#2277] [Everyday usage]
I downloaded pgroutin from homebrew with postgis and postgres as dependencies.

When I tried to create extension of pgrouting after creating with postgis, this problem shows up:

ERROR:  incompatible library "/usr/local/lib/postgresql/libpgrouting-3.2.so": version mismatch
DETAIL:  Server is version 14, library is version 13.

What should I do?

**Comment by SMillerDev** [comment]
You can run `brew reinstall -s pgrouting` as a temporary workaround until I have time to make a pull request that detects and fixes this.

**Reply by danpanetta** [reply]
Thanks a lot for your help

**Reply by SMillerDev** [reply]
The pull request should be merged by now. So you can also do `brew upgrade` to get the fix.
